.svglintrc.js

```diff
@@ -1,14 +1,21 @@
+if (Array.prototype.flat === undefined) {
```

Comment:
Minor thing, but could you move this right before the `if (updateIgnoreFile) {` line?

Reply:
Sure!